include/pet_tools.h

```diff
@@ -66,6 +67,7 @@ double calculate_net_radiation_W_per_sq_m(pet_model *model)
     saturation_water_vapor_partial_pressure_Pa=calc_air_saturation_vapor_pressure_Pa(model->surf_rad_forcing.air_temperature_C); 
 
   actual_water_vapor_partial_pressure_Pa=model->surf_rad_forcing.relative_humidity_percent/100.0*
+                                         // FIXME: REVIEWER: what should the default value be?
                                          saturation_water_vapor_partial_pressure_Pa;
```

Comment:
What should the default value be for `saturation_water_vapor_partial_pressure_Pa` if   `model->pet_options.yes_aorc==1` (expand the code up to see the preceding `if` block for context)?